app/exporters/response_exporter.rb

```diff
@@ -64,8 +64,8 @@ def sort_and_add_default_header_fields(headers)
     def response_hash(response)
       hsh = {
         'response_id' => response.id,
-        'filled_out_by_id' => response.filled_out_by.external_identifier,
-        'filled_out_for_id' => response.filled_out_for.external_identifier,
+        'filled_out_by_id' => response&.filled_out_by&.external_identifier,
+        'filled_out_for_id' => response&.filled_out_for&.external_identifier,
```

Comment:
Die eerste & na response mag weg. Dan zouden al die andere regels ook fout moeten gaan namelijk. We roepen dit nooit aan met een response die nil is.